zipline/algorithm.py

```diff
@@ -247,10 +259,14 @@ def __init__(self, *args, **kwargs):
 
         self._most_recent_data = None
 
+        self.initialized = False
+
+        # Register any default TradingControl objects
+        self.register_trading_control(ExpiredSID())
```

Comment:
@ssanderson Does it make sense to register this trading control in TradingAlgorithm.__init__?